src/lightning_app/cli/lightning_cli.py

```diff
@@ -337,6 +339,75 @@ def stop() -> None:
 _main.add_command(create)
 
 
+@_main.command("ssh")
+@click.option(
+    "--app-id",
+    "app_id",
+    type=str,
+    default=None,
+    required=False,
+)
```

Comment:
In the PR description you have the example where inquirer asks for the app name to connect to. 
I would suggest we also make it available here. Using the name seems the most intuitive to me, especially because the name is displayed in many places (CLI when you start app, in the admin front and center, etc.).

Reply:
fair point. I've reworked this and the docs to accept `app-name` instead.